near/src/config.rs

```diff
@@ -675,10 +708,20 @@ fn state_records_account_with_key(
 }
 
 /// Official TestNet configuration.
-pub fn testnet_genesis() -> GenesisConfig {
-    let genesis_config_bytes = include_bytes!("../res/testnet.json");
+pub fn testnet_genesis(genesis_records: &str, genesis_config: &str) -> GenesisConfig {
+    let mut f = File::open(genesis_records).expect("Failed to open genesis records");
+    let mut genesis_records_bytes = Vec::new();
+    f.read_to_end(&mut genesis_records_bytes).expect("Failed to read genesis records");
```

Comment:
Heads up! There is a convenient method in Rust standard library: https://doc.rust-lang.org/stable/std/fs/fn.read_to_string.html

```rust
std::fs::read_to_string("address.txt")?
```